### PR TITLE
Implement ``GetAvatarByUserID``

### PR DESCRIPTION
**Pull request by ghost**
This fixes previews being broken in buddy list in WoJS.

### CODE REVIEW

**Comment by ghost** [comment]
idk why it put previous commit

File: src/Services/BuddyService.cs [hunk 1]
```diff
@@ -25,20 +25,19 @@ public BuddyService(DBContext ctx, IOptions<ApiServerConfig> config, MessageServ
         public BuddyList GetBuddyList(Viking viking)
         {
             List<Schema.Buddy> buddies = new List<Schema.Buddy>();
-            foreach(Model.Buddy buddy in viking.Buddies)
-            {
-                Model.Buddy otherBuddy = ctx.Buddies.FirstOrDefault(e => e.OwnerID == buddy.BuddyID);
 
-                if (otherBuddy == null) continue;
+            List<Model.Buddy> vikingBuddies = ctx.Buddies.Where(e => e.OwnerID == viking.Id).ToList(); // relations aren't right, getting around it
```
Comment (rpaciorek) [inline src/Services/BuddyService.cs:29]: so why don't fix relation instead of placing this workaround in the next places?

File: src/Schema/BadgeData.cs
Comment (rpaciorek) [inline src/Schema/BadgeData.cs:1]: Where is it used?

Comment (ghost) [reply src/Schema/BadgeData.cs:1]: ``GetBadge`` but thats not in this PR atm

Comment (rpaciorek) [reply src/Schema/BadgeData.cs:1]: I would ask you not to add code to PR that is not used or does not work.

**Pull request closed without merging**